src/components/PostList.tsx

```diff
@@ -26,76 +34,58 @@ const PostList = ({ data, user }: PostListProps): JSX.Element => {
-              className="flex justify-start text-xs sm:text-xl text-grey transition-all duration-200 w-16 sm:w-24"
-            >
-              <div className="cursor-pointer flex justify-start items-center hover:text-saucyRed transition-all duration-200">
-                <FaArrowAltCircleUp aria-hidden="true" className="mr-1"/>
-                <p className="font-bold">{votes}</p>
-              </div>
-            </button>
-
-            <a
-              className="flex justify-start  text-xs sm:text-xl text-grey transition-all duration-200 w-16 sm:w-24"
-              href={getRepoIssuesLink(data.full_name)}
-              title={`Visit ${data.full_name} issues`}
-              target="_blank"
-              rel="noopener"
-            >
-              <div className="cursor-pointer flex justify-start items-center hover:text-saucyRed transition-all duration-200">
-                <FaDotCircle aria-hidden="true" className="mr-1"/>
-                {data.issues && <p className="font-bold">{humanizeNumber(data.issues)}</p>}
-              </div>
-            </a>
-
+        <div className='flex-1'>
             <a
-              className="flex justify-start  text-xs sm:text-xl text-grey transition-all duration-200 w-16 sm:w-24"
-              href={getRepoLink(data.full_name)}
-              title={`Add a star to ${data.full_name}`}
+              href={getRepoLink(full_name)}
+              title={`Visit ${full_name}`}
               target="_blank"
               rel="noopener"
-            >
-              <div className="cursor-pointer flex justify-start items-center hover:text-saucyRed transition-all duration-200">
-                <FaStar aria-hidden="true" className="mr-1"/>
-                {data.stars && <p className="font-bold">{humanizeNumber(data.stars)}</p>}
-              </div>
+              >
+                <p className='text-[14px] text-textGrey'>{full_name}</p>
+                <p className='text-[16px] text-textGrey'>{description}</p>
             </a>
-          </div>
+            <div className='flex gap-x-[16px] mt-[16px]'>
+                <div className='flex gap-[5px] items-center text-textGrey'>
+                    <img className='w-[16px]' src={issueIconGrey} alt="issues"/>
+                    <p className='text-[14px]'>{humanizeNumber(issues)}</p>
+                </div>
+                <div className='flex gap-[5px] items-center text-textGrey'>
+                    <img className='w-[16px]' src={starIconGrey} alt="stars"/>
+                    <p className='text-[14px]'>{humanizeNumber(stars)}</p>
+                </div>
+                <div className='flex gap-x-[5px]'>
+                    {
+                      contributions.slice(0, 5).map(({contributor, last_merged_at}) => (
+                        <div className='w-[24px] h-[24px] overflow-hidden rounded-full -mr-[15px] transition-all duration-300 hover:mr-0 hover:scale-105'>
```

Comment:
```suggestion
                        <div className='w-[24px] h-[24px] overflow-hidden rounded-full -mr-[15px] transition-all duration-300 hover:mr-0 hover:scale-150'>
```

Switch up the hover:scale to 150